ui/jobs/buff_tracker.ts

```diff
@@ -277,26 +277,29 @@ export class BuffTracker {
         sortKey: 0,
       },
       offguard: {
-        gainAbility: kAbility.OffGuard,
```

Comment:
Should `gainAbility` be removed in general or are there other uses still?

Reply:
I think it is better to keep it remained, 'cause we don't know if there are skills that can only be handled by `gainAbility` added in the new patch.

Reply:
I think we will put it to use again in the next PR _(:3rz)_